### DISCUSSION TITLE
What's your meeting workload like?

### DISCUSSION THREAD

**Discussion by ileitch** [MobileNativeFoundation/discussions#149] [General]
Hey all,

I'm curious to hear from other mobile engineers (specifically engineers, not managers) what your average weekly meeting workload is like?

Personally I've 8-10 hours of meetings per week. That includes everything I've committed to, plus things like All Hands. Spending on average a entire day of my week in meetings feels excessive to me, but I wonder how that compares to others?

**Comment by piannaf** [comment]
I'm a manager, so I won't tell you how many (many many) hours I spend in meetings per week (thanks for calling that out 😰)

I just analyzed the hours per week for each of our devs for the past 2.5 months and the only devs who averaged >8h/w are our most senior devs who get brought into meetings for multiple clients and other internal meetings more often than other devs. We have one client that is more meeting-happy than other clients, and that is proven by the data: devs on that project average between 7 and 7.5h/w in meetings while devs on our other clients average <7h/w.

(Like you, I also included various all-hands (non-client) meetings in the data)

https://github.com/aronwoost/google-calendar-hours can help everyone with Google Calendar discover where there time is going

If you are on a Business Plus or Enterprise plan, Google Calendar also has [Time Insights](https://support.google.com/calendar/answer/10738043?hl=en)

**Reply by ileitch** [reply]
> I'm a manager, so I won't tell you how many (many many) hours I spend in meetings per week (thanks for calling that out 😰)

Ah, now I feel bad for excluding managers 😅. Feel free to share!

> I just analyzed the hours per week for each of our devs for the past 2.5 months and the only devs who averaged >8h/w are our most senior devs who get brought into meetings for multiple clients and other internal meetings more often than other devs. We have one client that is more meeting-happy than other clients, and that is proven by the data: devs on that project average between 7 and 7.5h/w in meetings while devs on our other clients average <7h/w.

Many thanks for investigating, this is very insightful.

**Reply by piannaf** [reply]
Oh sorry, I didn't mean to make you feel bad. I was confirming, in a snarky way, that it's a good idea to exclude managers. My reaction was more about getting reminded that I have so many meetings 😅 

For the same time-frame, I averaged close to 17h/w. Some of those meetings I skip sometimes, and some of those meetings go long sometimes so it's probably still a good approximation. (About a month before that timeframe, hired someone to delegate some of my responsibilities. Before that, my meeting load too often got close to 30h/w. That'll probably happen again soon as I take on more responsibilities that require meetings... then I may need to hire again)

**Comment by abdulrahim46** [comment]
At my current role as a Lead Engineer. I'm spending around 4-5 hrs per week on meetings, it's nothing but still it can be improve to 2hrs per week I think.
